pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -282,6 +286,11 @@ type CanaryStrategy struct {
 	// ScaleDownDelayRevisionLimit limits the number of old RS that can run at one time before getting scaled down
 	// +optional
 	ScaleDownDelayRevisionLimit *int32 `json:"scaleDownDelayRevisionLimit,omitempty" protobuf:"varint,12,opt,name=scaleDownDelayRevisionLimit"`
+	// AbortScaleDownDelaySeconds adds a delay before scaling down the canary pods when update
+	// is aborted for canary strategy using replicas of setCanaryScale.
+	// Default is 0, meaning canary pods are not scaled down.
```

Comment:
Let's add a comment stating that this field is only used with traffic routing and not applicable for basic canary.